File: Herencia/ejercicio 2/canguro.py
from animal import Animal

class Canguro(Animal):
    def __init__(self, nombre: str, edad: int):
        super()._init_(nombre, edad)

    def desplazarse(self):
        print(f"{self.nombre} salta con sus patas fuertes.")

    def __str__(self):
        return f"Canguro(nombre={self.nombre}, edad={self.edad})"

File: Herencia/ejercicio 2/Canguro.java
package herencia;

public class Canguro extends Animal {

    public Canguro(String nombre, int edad) {
        super(nombre, edad);
    }

    @Override
    public void desplazarse() {
        System.out.println(nombre + " salta con sus patas fuertes.");
    }
}